lib/event_sourcery/postgres/schema.rb

```diff
@@ -1,7 +1,7 @@
 module EventSourcery
   module Postgres
     module Schema
-      extend self
+      module_function
```

Comment:
`module_function` reads a bit better initially to me. I also believe that `module_function` methods when their encompassing module gets included into a class those methods do not become public to that class. Unless thats what we want?